tox.ini

```diff
@@ -44,7 +46,10 @@ setenv =
   COVERAGE_PROCESS_START={toxinidir}/.coveragerc
   FULL_COVERAGE=true
   nonhdfs: NOSE_WITH_DOCTEST=1
+  hdf5: NOSE_ATTR=hdf5
 commands =
+  # ensure pandas is installed after cython
```

Comment:
Pandas needs cython installed to build from source because there is no py33 wheel on PyPi. See https://pypi.python.org/pypi/pandas